### PR TITLE
Protect: Prepare changelog for 1.0.1

### PR DESCRIPTION
**Pull request by miguelxpn**
#### Changes proposed in this Pull Request:

* This prepares the changelogs for the 1.0.1 point release of Jetpack Protect.

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Nothing to test really.

### CODE REVIEW

File: projects/plugins/protect/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.1 - 2022-07-07
+### Added
+- Added new two FAQ entries
```
Comment (sdixon194) [inline projects/plugins/protect/CHANGELOG.md]:
```suggestion
- Added two new FAQ entries
```

File: projects/plugins/protect/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
...
+- Protect: record even just after the site is registered
+
+### Changed
+- Moved normalization of the Protect status report data to the server side.
+- Renaming master to trunk.
+- Renaming `master` references to `trunk`
```
Comment (sdixon194) [inline projects/plugins/protect/CHANGELOG.md]:
```suggestion
- Renamed`master` references to `trunk`
```
Can probably keep this line and remove the previous `renaming master to trunk`

File: projects/plugins/protect/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
...
+- Reorder JS imports for `import/order` eslint rule.
+
+### Fixed
+- Fixed bug that would not display Core vulnerabilities, remove legacy code and documentation and add new docs on the debug helper plugin.
```
Comment (sdixon194) [inline projects/plugins/protect/CHANGELOG.md]:
```suggestion
- Fixed bug that would not display Core vulnerabilities.
- Removes legacy code and documentation and adds new docs for the debug helper plugin.
```

**Review by sdixon194 — APPROVED** [review]

**Comment by coder-karen** [comment]
r248854-wpcom

**Pull request merged**